edilkamin/api.py

```diff
@@ -201,3 +201,148 @@ def set_target_temperature(token: str, mac_address: str, temperature: int) -> st
+    info = device_info(token, mac_address)
+    fan_number_confiuguration = info["nvm"]["installer_parameters"]["fans_number"]
+    
+    if fan_number_confiuguration >= 3:
+        """
+        Set fan 3 speed.
+        Return response string e.g. "Command 0123456789abcdef executed successfully".
+        """
+        return mqtt_command(token, mac_address, {"name": "fan_3_speed", "value": fan_3_speed})
+
+
+def device_info_get_airkare(info: typing.Dict) -> int:
+    """Get airkare status from cached info."""
+    return info["status"]["commands"]["airkare_function"]
+
+
+def get_airkare(token: str, mac_address: str) -> int:
+    """Get airkare status."""
+    info = device_info(token, mac_address)
+    return device_info_get_airkare(info)
+
+
+def set_airkare(token: str, mac_address: str, airkare: bool) -> str:
+    """
+    Set airkare.
+    Return response string e.g. "Command 0123456789abcdef executed successfully".
+    """
+    return mqtt_command(token, mac_address, {"name": "airkare_function", "value": airkare})
+
+
+def device_info_get_relax_mode(info: typing.Dict) -> int:
+    """Get relax mode status from cached info."""
+    return info["nvm"]["user_parameters"]["is_relax_active"]
+
+
+def get_relax_mode(token: str, mac_address: str) -> int:
+    """Get relax mode status."""
+    info = device_info(token, mac_address)
+    return device_info_get_relax_mode(info)
+
+
+def set_relax_mode(token: str, mac_address: str, relax_mode: bool) -> str:
+    """
+    Set relax mode.
+    Return response string e.g. "Command 0123456789abcdef executed successfully".
+    """
+    return mqtt_command(token, mac_address, {"name": "relax_mode", "value": relax_mode})
+
+
+def device_info_get_manual_power_level(info: typing.Dict) -> int:
+    """Get manual power level value from cached info."""
+    return info["nvm"]["user_parameters"]["manual_power"]
+
+
+def get_manual_power_level(token: str, mac_address: str) -> int:
+    """Get manual power level value."""
+    info = device_info(token, mac_address)
+    return device_info_get_relax_mode(info)
+
+
+def set_manual_power_level(token: str, mac_address: str, manual_power_level: int) -> str:
+    """
+    Set manual power level value.
+    Return response string e.g. "Command 0123456789abcdef executed successfully".
+    """
+    return mqtt_command(token, mac_address, {"name": "power_level", "value": manual_power_level}) 
+
+
+def device_info_get_standby_mode(info: typing.Dict) -> int:
+    """Get standby mode status from cached info."""
+    return info["nvm"]["user_parameters"]["is_standby_active"]
+
+
+def get_standby_mode(token: str, mac_address: str) -> int:
+    """Get standby mode status."""
+    info = device_info(token, mac_address)
+    return device_info_get_standby_mode(info)
+
+
+def set_standby_mode(token: str, mac_address: str, standby_mode: bool) -> str:
+    info = device_info(token, mac_address)
+    is_auto = info["nvm"]["user_parameters"]["is_auto"]
+    
+    if is_auto == True:
```

Comment:
https://peps.python.org/pep-0008/#programming-recommendations
> Don’t compare boolean values to True or False using ==:
```suggestion
    if is_auto:
```